GOVERNANCE.md

```diff
@@ -11,7 +11,7 @@ developed openly and hosted in public GitHub repositories under the
 [GitHub organization](https://github.com/arviz-devs). Examples of
 Project Software include the ArviZ code and the Documentation, etc. The Services run by the
 Project consist of public websites and web-services that are hosted
-at [https://arviz-devs.github.io/arviz/](https://arviz-devs.github.io/arviz/)
```

Comment:
commenting here because it felt the most appropriate place. The governance doc should be converted to a sentence and a link to the governance section in arviz.org, and the election folder should be deleted too (it is already in arviz-project repo)